.github/workflows/scripts/kinds/verify-kinds.go

```diff
@@ -40,56 +33,13 @@ const (
 // If kind names are given as parameters, the script will make the above actions only for the
 // given kinds.
 func main() {
-	var corek []kindsys.Kind
-	var compok []kindsys.Composable
 
 	kindRegistry, err := NewKindRegistry()
 	defer kindRegistry.cleanUp()
 	if err != nil {
 		die(err)
 	}
 
-	// Search for the latest version directory present in the kind-registry repo
-	latestRegistryDir, err := kindRegistry.findLatestDir()
-	if err != nil {
-		die(fmt.Errorf("failed to get latest directory for published kinds: %s", err))
-	}
-
-	errs := make([]error, 0)
-
-	// Kind verification
-	for _, kind := range corekind.NewBase(nil).All() {
-		name := kind.Props().Common().MachineName
-		err := verifyKind(kindRegistry, kind, name, "core", latestRegistryDir)
-		if err != nil {
-			errs = append(errs, err)
-			continue
-		}
-
-		corek = append(corek, kind)
-	}
-
-	for _, pp := range corelist.New(nil) {
-		for _, kind := range pp.ComposableKinds {
-			si, err := kindsys.FindSchemaInterface(kind.Def().Properties.SchemaInterface)
-			if err != nil {
-				errs = append(errs, err)
-				continue
-			}
-
-			name := strings.ToLower(fmt.Sprintf("%s/%s", strings.TrimSuffix(kind.Lineage().Name(), si.Name()), si.Name()))
-			err = verifyKind(kindRegistry, kind, name, "composable", latestRegistryDir)
-			if err != nil {
-				errs = append(errs, err)
-				continue
-			}
-
-			compok = append(compok, kind)
-		}
-	}
-
-	die(errs...)
-
 	if _, set := os.LookupEnv("CODEGEN_VERIFY"); set {
```

Comment:
Should this go as well? Since all the verification stuff is now gone? Which makes me think that this file could also be renamed to something more appropriate.